guides/source/debugging_rails_applications.md

```diff
@@ -109,7 +109,7 @@ It can also be useful to save information to log files at runtime. Rails maintai
 
 Rails makes use of the `ActiveSupport::Logger` class to write log information. Other loggers, such as `Log4r`, may also be substituted.
 
-You can specify an alternative logger in `environment.rb` or any other environment file, for example:
+You can specify an alternative logger in `config/environment.rb` or any other environment file, for example:
 
 ```ruby
 Rails.logger = Logger.new(STDOUT)
```

Comment:
And below should be `Rails.application.config.logger` but I think that may not work so better to check.